router/src/hooks.rs

```diff
@@ -216,3 +216,25 @@ pub(crate) fn use_is_back_navigation() -> ReadSignal<bool> {
     let router = use_router();
     router.inner.is_back.read_only()
 }
+
+/// Resolves a redirect location to an (absolute) URL.
+pub(crate) fn resolve_redirect_url(loc: &str) -> Option<Url> {
+    let origin = match window().location().origin() {
```

Comment:
Would it be ok to `unwrap()` or `unwrap_throw()` here? Is the `.origin()` call always expected to succeed?